action.yaml

```diff
@@ -60,6 +60,9 @@ inputs:
   configuration:
     type: string
     description: Autobuild configuration
+  artifact-name:
```

Comment:
Duplicated artifact names are, currently, only a concern of the tests in this repository itself. However, it may be generally useful for consumers of the action to be able to explicitly set artifact name.